Firestore/Example/Tests/Integration/API/FIRFieldsTests.mm

```diff
@@ -255,52 +255,7 @@ - (void)testTimestampsInSnapshots {
   XCTAssertEqualObjects(truncatedTimestamp, timestampFromData);
   XCTAssertEqualObjects(timestampFromSnapshot, timestampFromData);
 }
-@end
```

Comment:
In the Android PR that I started working on, I renamed the previous test to `testTimestampsAreTruncated`, mostly just get rid of the 'timestampsInSnapshots" term from the codebase. Let me know what you end up deciding since the test names should end up matching.

Reply:
I renamed to match your Android change.